windows_containers/windows-containers-release-notes-2-x.adoc

```diff
@@ -64,6 +64,15 @@ This release adds the following new features and improvements.
 
 Image pull time-out has been increased to 30 minutes.
 
+[id="wmco-2-0-1-autoscaling-windows-instances"]
+==== Autoscaling for Windows instances
+
+Cluster autoscaling is now supported for Windows instances. You can complete the following actions for Windows nodes:
+
+* Define and deploy a xref:../machine_management/applying-autoscaling.adoc#configuring-clusterautoscaler[cluster autoscaler].
+* Create a Windows node using a machine set.
+* Define and deploy a xref:../machine_management/applying-autoscaling.adoc#configuring-machineautoscaler[machine autoscaler], referencing a Windows machine set.
```

Comment:
My ignorance. Is there a difference between a _machine set_ and a _Windows machine set_?

Should we link to machine set in the second bullet for consistency, assuming there is a proper location to link to?

Reply:
There is a difference between a machine set and a Windows machine set. I will add the link there for consistency :+1: Thanks!